keras_retinanet/models/retinanet.py

```diff
@@ -273,13 +298,30 @@ def retinanet(
     if submodels is None:
         submodels = default_submodels(num_classes, num_anchors)
 
-    C3, C4, C5 = backbone_layers
+    if pyramid_levels is None:
+        pyramid_levels = [3, 4, 5, 6, 7]
+
+    C2 = None
+
+    C3 = backbone_layers['C3']
+    C4 = backbone_layers['C4']
+    C5 = backbone_layers['C5']
+
+    if 2 in pyramid_levels:
+        if 'C2' in backbone_layers:
+            C2 = backbone_layers['C2']
+        else:
+            raise ValueError('C2 not provided by backbone model. Cannot create P2 layers.')
```

Comment:
We make the habit of putting "user readable strings" in double quotes, like this one.

Reply:
fixed.